is_core/generic_views/mixins.py

```diff
@@ -64,21 +114,10 @@ def get_tabs(self):
         return self.tabs
 
     def get_tab_menu_items(self):
-        from is_core.menu import LinkMenuItem
-
-        menu_items = []
-        for tab in self.get_tabs():
-            if len(tab) == 2:
-                tab_title, tab_url = tab
-                is_active = self.request.path == tab_url
-            else:
-                tab_title, tab_url, is_active = tab
-
-            menu_items.append(LinkMenuItem(tab_title, tab_url, active=is_active))
-        return menu_items
+        return [tab.get_menu_link_item_or_none(self) for tab in self.get_tabs() if tab.can_show(self)]
 
-    def get_context_data(self, form=None, **kwargs):
-        context_data = super(TabsViewMixin, self).get_context_data(form=form, **kwargs)
+    def get_context_data(self, **kwargs):
+        context_data = super(TabsViewMixin, self).get_context_data(**kwargs)
```

Comment:
tohle byla chyba jak prase generoval jsem tam zbytečnou hodnotu která měla none do context_data házelo mi to pak chybu v KNJ a mohl jsem se zbláznit abych to našel